### ISSUE TITLE
validation result

### ISSUE THREAD

**Issue by stone-cloud** [zh320/medical-segmentation-pytorch#1]
Great work! Thank you for your contribution. I used this framework to train the model, but during the final validation after training, the best validation score was printed twice with different results. Which one is the final result? What do these two scores represent?

**Comment by zh320** [comment]
Hi,

I am assuming you were using both the validate and test set. If you look at the following lines, you could find that the first result is for the validate set while the second one is for the test set.

Best,

https://github.com/zh320/medical-segmentation-pytorch/blob/79b02534f8e55d70fe4194f271bbaa58f264a47f/core/base_trainer.py#L117-L120
